core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -475,7 +475,52 @@ class DAGSchedulerSuite
     assert(results === Map(0 -> 42, 1 -> 43))
     assertDataStructuresEmpty()
   }
+  
+  test("Test taskAbort after multiple stage failures.") {
```

Comment:
It's hard to understand what this test does from the name. Change to "Multiple consecutive stage failures should lead to stage being aborted"?